Makefile

```diff
@@ -42,6 +42,12 @@ $(PACKAGE_SCRIPTS) \
 --config-files "/etc/$(BINARY)/$(CONFIG_FILE)"
 endef
 
+VERSION_LDFLAGS:= \
+  -X $(IMPORT_PATH)/vendor/github.com/prometheus/common/version.Branch=$(BRANCH) \
+  -X $(IMPORT_PATH)/vendor/github.com/prometheus/common/version.BuildDate=$(DATE) \
+  -X $(IMPORT_PATH)/vendor/github.com/prometheus/common/version.Revision=$(COMMIT) \
+  -X $(VERSION_PATH)=$(VERSION)-$(ITERATION)
```

Comment:
I'd be fine converting this too, remove the local global variable and use `version.Version` directly. Pending your "thought" on my other comment.

Reply:
I think it's fine this way for now.